src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,325 @@
+
+    ```astro title="src/pages/es/index.astro"
+    ---
+    import { getRelativeLocaleUrl } from 'astro:i18n';
+    const aboutURL=getRelativeLocaleUrl("es", "about");
+    ---
+
+    <a href="/es/get-started/">¡Vamos!</a>
+    <a href={getRelativeLocaleUrl('es', 'blog')}>Blog</a>
+    <a href={aboutURL}>Acerca</a> 
+    ```
+
+### `defaultLocale` and `locales`
+
+Both a default language ([`defaultLocale`](en/reference/configuration-reference/#experimentali18ndefaultlocale)) and a list of all supported languages ([`locales`](en/reference/configuration-reference/#experimentali18nlocales)) must be specified in your `i18n` routing configuration.
+
+Each language must be a string (e.g. `"fr"`, `"pt_BR"`), but no particular language format or syntax is enforced while this feature is still experimental and under development. This may be subject to change in future versions.
+
+Your `/[lang]/` folder names must match exactly the `locales` in the list, and every supported language must have its own localized folder. 
+
+Depending on your deploy host, you may discover transformations in URL paths, so check your deployed site to determine the best syntax for your project.
+
+### Routing strategy 
+
+You can choose whether or not pages in your `defaultLocale` should be prefixed with their `/lang/` in their URL path. Every other supported locale will automatically be prefixed.
+
+#### `'prefix-other-locales'`
+
+```js title="astro.config.mjs" ins={7}
+import { defineConfig } from "astro/config"
+export default defineConfig({
+  experimental: {
+    i18n: {
+      defaultLocale: "en",
+      locales: ["es", "en", "fr"],
+      routingStrategy: "prefix-other-locales"
+    }
+  }
+})
+```
+
+This is the **default** value. With this option, URLs in your default language will **not** have a `/[lang]/` prefix.
+
+For example, if `i18n.defaultLocal: "en"` is configured: 
+
+- `src/pages/en/blog.astro` will produce the route `example.com/blog/` 
+- `src/pages/fr/blog.astro` will produce the route `example.com/fr/blog/` 
+- If there is no file at `src/pages/es/blog.astro`, then the route `example.com/es/blog/` will 404 unless you specify a [fallback strategy](#fallback).
+
+
+#### `'prefix-always'`
+
+```js title="astro.config.mjs" ins={7}
+import { defineConfig } from "astro/config"
+export default defineConfig({
+  experimental: {
+    i18n: {
+      defaultLocale: "en",
+      locales: ["es", "en", "fr"],
+      routingStrategy: "prefix-always"
+    }
+  }
+})
+```
+
+With this option, all routes will have their locale code (prefix) in their URL.
+
+- `src/pages/[lang]/blog.astro` will all reproduce routes of the form `example.com/[locale]/blog/`.
```

Comment:
Not sure if this "all" has a reason here... doesn't sound really right to me.

```suggestion
- `src/pages/[lang]/blog.astro` will reproduce routes of the form `example.com/[locale]/blog/`.
```

Reply:
Gotcha. I'll rephrase!